src/Services/WorkItemMutationWorkflow.php

```
@@ -0,0 +1,100 @@
+            ) ?: null,
+        ];
+
+        if (array_key_exists('team_assigned', $item)) {
+            $updateData['team_assigned'] = $owner;
+        }
+
+        if ($newEstimatedSprints !== '') {
+            $updateData['estimated_sprints'] = (int) $newEstimatedSprints;
+        }
+
+        HLWorkItem::updateForOrg($this->db, $itemId, $orgId, $updateData);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\StratFlow\Models\HLWorkItem' in method 'updateFromPayload'. (undefined)

(StaticAccess)